http-wrapper/src/server.rs

```diff
@@ -483,4 +485,39 @@ where
         .untuple_one()
         .and_then(encrypt_and_generate_response::<IM, OM>)
         .boxed()
+
 }
+
+//This method to keep things more inline with the framework
+pub fn ov_request_filter<UDT, F, FR>( 
+    user_data: UDT,
+    handler: F,
+) -> warp::filters::BoxedFilter<(warp::reply::Response,)>
+where 
+    UDT: Clone + Send + Sync + 'static,
+    //option 1 works
+    F: Fn(HeaderValue,UDT) -> FR + Clone + Send + Sync + 'static, 
+    // option 2 works
+    // F: Fn(HeaderValue,&[u8],UDT) -> FR + Clone + Send + Sync + 'static,
+    FR: futures::Future<Output = Result<warp::reply::Response, warp::Rejection>> + Send,
+{
+    let baseurl = warp::path("management").and(warp::path("v1"));
+    baseurl        
+        .and(warp::post())
+        .and(warp::path("ownership_voucher"))
+        .and(warp::path::end())
+        .and(warp::header::exact("Authorization","Bearer"))
+        .and(warp::header::value("X-Number-Of-Vouchers"))
+        .and(warp::body::content_length_limit(1024 * 32))
+        // option 1 works
+        .map(move |c| (c,user_data.clone()))
+        
+        // option 2 doesnt line: 521 complains, and I need to parse the body for the ownership voucher
+        // .and(warp::body::aggregate())
+        // .map(move |c,body| (c,body,user_data.clone()))
```

Comment:
@puiterwijk This method will put things more in line with the existing framework, but not able to parse the body

Reply:
What is the error you get if you use this case?
I'm going to guess it's related to the `&[u8]` lifetime, or the fact that that's not an `impl Buf`?

Reply:
Geez!! the reply notification came to my gmail, which I didn't notice, until today..!!!!!!